On branch feature/set-webpack

estebanmual opened the pull request:
In this milestone I: 
- set up webpack
- set up linters

estebanmual committed:
On branch feature/set-webpack
	new file:   .eslintrc.json
	new file:   .gitignore
	new file:   .hintrc
	new file:   .stylelintrc.json
	new file:   linters.yml
	new file:   package-lock.json
	new file:   package.json
	new file:   src/app/app.js
	new file:   src/app/utils/files.md
	new file:   src/assets/assets.md
	new file:   src/index.js
	new file:   src/main.css
	new file:   src/template.html
	new file:   src/vendor.js
	new file:   webpack.common.js
	new file:   webpack.dev.js
	new file:   webpack.prod.js
	Set up webpack and linters.

yersel500 reviewed (changes requested):
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽  @estebanmual 

Good job working on the project so far! 
✔️ Good installation of webpack.

However, there are some issues that you need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/l3JDhOnR017YfwIeY/giphy.gif)

## Required Changes ♻️
- [x] The linters aren't working well. Please check it and fix it before moving to the next steps.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

estebanmual committed:
On branch feature/set-webpack
	new file:   .github/workflows/linters.yml
	Add linters.yml file to show them in github

estebanmual committed:
linters errors

estebanmual committed:
linters errors

yersel500 reviewed (approved):
# STATUS: APPROVED :green_circle: :clap: 

Hi @estebanmual , 

Great work on making the changes requested 👏 

Your setup is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Well structured webpack 💯 
- You set up linter files and no errors. 💯 

![well-1](https://user-images.githubusercontent.com/64173189/128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
